tests/vmi_multus_test.go

```diff
@@ -1240,3 +1258,35 @@ func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmiPod *
 
 	return nil
 }
+
+func defaultCloudInitNetworkData() string {
+	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should success creating default cloud init network data for SRIOV")
+	return networkData
+}
+
+func cloudInitNetworkDataWithStaticIPsByMac(macAddress, ipv4Address string) string {
```

Comment:
The DHCP was not mentioned here and I am not sure if it absence is not an implicit "enabled". Could you please confirm from inside the VM?

Reply:
originally it was not enabled either, I will check though just to be sure.

Reply:
For IPv4 I see `static` 
```bash
[root@testvmi6pqv6lnzqh2j4wkngpz5bwpqffbxpj9lt8gkczf9xt24tv6j fedora]# nmcli c show "System eth1" |grep ipv4.method
ipv4.method:                            auto
```

For IPv6 Is auto, I suppose we don't care here.
```bash
[root@testvmi6pqv6lnzqh2j4wkngpz5bwpqffbxpj9lt8gkczf9xt24tv6j fedora]# nmcli c show "System eth1" |grep ipv6.method
ipv6.method:                            auto
```

Reply:
Ok, thanks!